### DISCUSSION TITLE
【灌水】以太适性自测结果是这个分数，我能去空洞里搬砖吗？

### DISCUSSION THREAD

**Discussion by ZnS-1145** [share121/inter-knot#1086] [其他]
![微信图片_20240810222212](https://github.com/user-attachments/assets/ec0288b6-c0e4-4c75-83c8-34348bd57047)
本人学生，高中还没毕业。老爸因为工作原因买了以太适性自测卡，我出于好奇就随便也跟着测了一下，结果拿了个80分。老爸当时就从椅子上跳起来，逼着我重新测了一次...据说50分是能从事空洞相关工作的及格线，80分是很高的分数了，这是不是意味着我也可以去空洞里当搬运工了？

**Comment by ShallowDream12321** [comment]
之前很多人都测过结果都很高 之后爆出这批是假货 有的去了空洞就再也没回来

**Comment by Mu-7593** [comment]
真羡慕你们以太适性高的人，有更多选择，像我这种普通人只能老老实实的打工

**Comment by lllliufeng** [comment]
有这适性当什么搬运工啊，协会肯定请你去掉王牌调查员啊

**Comment by APEX-114514** [comment]
你该不是故意在秀吧，都80分了还当什么搬运工?未来的王牌调查员就是你了啊。

**Comment by liyiwuhua** [comment]
这位同学，你好，Liyiwuhua院诚邀你的加入！Liyiwuhua建校历史悠久，师资雄厚，以培养高端研究人才为己任。若确认你的以太适应分数无误，我们热忱邀请你加入以太适应班，无门槛办理转学手续，学费减半并设有丰厚奖学金。在哪里你一定能踏上理想的道路，发挥你非比寻常的才能。详情已发送私信，若有兴趣请查收和回复。

**Comment by Modog1145** [comment]
Liyiwuhua院校的速度好快啊，是不是专门邀了个老师在绳网刷帖子啊。
